### DISCUSSION TITLE
VueStorefront integration

### DISCUSSION THREAD

**Discussion by olivermrbl** [medusajs/medusa#2702] [RFC]
Discussion is dedicated to sharing ideas and thoughts around a potential VueStorefront integration. 

If anyone is already working on a solution, please let us know below - we are more than happy to assist you in whatever way is possible

Below are relevant links related to the integration:
- [VueStorefront Discord](https://discord.com/channels/876835651130097704/920640417433808899/945420752205787176)
- [Community repository for the integration](https://discord.com/channels/876835651130097704/920640417433808899/948612628652097547)
- [VueStorefront Documentation](https://docs.vuestorefront.io/v2/getting-started/introduction.html)
- [Medusa Documentation](https://docs.medusajs.com)

**Comment by ymaheshwari1** [comment]
Hi @olivermrbl 

I am working on the integration between medusa and VSF, currently, maintaining the integration here: https://github.com/ymaheshwari1/medusa-1/tree/develop

The integration is in its initial phase.
